frontend/css_and_js.py

```diff
@@ -1,37 +1,32 @@
 from os import path
 
+
 def readTextFile(*args):
     dir = path.dirname(__file__)
     entry = path.join(dir, *args)
     with open(entry, "r", encoding="utf8") as f:
         data = f.read()
     return data
 
+
 def css(opt):
     styling = readTextFile("css", "styles.css")
     if not opt.no_progressbar_hiding:
         styling += readTextFile("css", "no_progress_bar.css")
     return styling
 
+
 def js(opt):
     data = readTextFile("js", "index.js")
     data = "(z) => {" + data + "; return z ?? [] }"
     return data
 
-# Wrap the typical SD method call into async closure for ease of use
-# If you call frontend method without wrapping
-# DONT FORGET to bind input argument if you need it: SD.with(x)
-def w(sd_method_call):
-    return f"async (x) => {{ return await SD.with(x).{sd_method_call} ?? x ?? []; }}"
-
-def js_move_image(from_id, to_id):
-    return w(f"moveImageFromGallery('{from_id}', '{to_id}')")
-
-def js_copy_to_clipboard(from_id):
-    return w(f"copyImageFromGalleryToClipboard('{from_id}')")
 
-def js_painterro_launch(to_id):
-    return w(f"Painterro.init('{to_id}')")
-
-def js_img2img_submit(prompt_row_id):
-    return w(f"clickFirstVisibleButton('{prompt_row_id}')")
+# Wrap the typical SD method call into async closure for ease of use
+# Supplies the js function with a params object
+# That includes all the passed arguments and input from Gradio: x
+def call_SD(sd_method, **kwargs):
```

Comment:
Should this be renamed callJS ? 
callSD is ambiguous (SD refers to a lot of things in this field)

Reply:
Could be a great place for 1 usage example as well

Reply:
SD is the global js object. The logic behind this is that the whole line in code will look like so:
`_js=call_SD('foo')` will translate in js into `SD.foo()`
I'd argue `_js=call_JS` is ambiguous because it's not telegraphed what where will that call go.

Reply:
It's no ambiguous in the python context, as _js is a python attribute, and call_JS is a python method, and what it does, it calling a JS function. 
Naming the global object SD is confusing to me, please address

Reply:
I can rename python's `call_SD` to `call_JS` no problem. What should be a global object named be then if not SD? GradioHelper?

Reply:
I agree with @codedealer it makes more sense to be named `call_SD`